library/cwm/src/lib/cwm/table.rb

```diff
@@ -109,6 +146,16 @@ def cell(*args)
       Yast::Term.new(:cell, *args)
     end
 
+    # helper to create a {TableItem}
+    # @param args content of item
```

Comment:
maybe add link for arg to TableItem#initialize?

Reply:
Done